src/internal/interfaces/Visualization.ts

```diff
@@ -261,11 +261,20 @@ export interface IUiConfig {
 }
 
 export interface IVisualizationProperties {
+    controls?: IVisualizationPropertiesControls;
+    sortItems?: AFM.SortItem[];
+}
+
+export interface IVisualizationPropertiesControls {
     // This can be anything depending on a visualization type
-    // perhaps consider adding: sortItems?: AFM.SortItem[]
```

Comment:
Is this comment style valid ?

Reply:
do we need type for this? Maybe any will be better.

Reply:
this comment is not valid, that is the reason why it is removed ;)

Reply:
sorry my fail was removed ok